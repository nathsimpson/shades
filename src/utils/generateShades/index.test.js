const { generateShades } = require('./index');

test(`expects correct Hue for ${name}`, () => {
  expect(generateShades('#fa6d01', 13, 40)).toMatchSnapshot();
});
