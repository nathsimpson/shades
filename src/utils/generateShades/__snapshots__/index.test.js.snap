// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects correct Hue for  1`] = `
Array [
  Object {
    "lightness": 160,
    "value": "#512300",
  },
  Object {
    "lightness": 220,
    "value": "#703101",
  },
  Object {
    "lightness": 290,
    "value": "#934001",
  },
  Object {
    "lightness": 360,
    "value": "#b75001",
  },
  Object {
    "lightness": 420,
    "value": "#d55d01",
  },
  Object {
    "lightness": 490,
    "value": "#f96c01",
  },
  Object {
    "lightness": 560,
    "value": "#fe8020",
  },
  Object {
    "lightness": 620,
    "value": "#fe913e",
  },
  Object {
    "lightness": 690,
    "value": "#fea662",
  },
  Object {
    "lightness": 760,
    "value": "#feba85",
  },
  Object {
    "lightness": 820,
    "value": "#ffcba4",
  },
  Object {
    "lightness": 890,
    "value": "#ffdfc7",
  },
]
`;
